sys/sys/proc.h

```diff
@@ -893,6 +893,11 @@ struct proc {
 						   permitted */
 #define	P2_REAPKILLED		0x00080000
 
+#define	P2_CHERI_REVOKE_ENABLE	0x40000000	/* Force enable revocation */
+#define	P2_CHERI_REVOKE_DISABLE	0x80000000	/* Force disable revocation */
```

Comment:
Why is this avoiding 0x80000000 for ELF_BSDF_* but using it here for P2_CHERI_REVOKE_DISABLE without concern?

Reply:
I changed the other one because AT_BSDFLAGS gets sign extended on 64-bit and looks ugly in `procstat auxv` output. Those bits aren't currently scarce so it seemed fine. These bits are relatively scarce so I stuck with 0x80000000, but could slide down one bit, particularly since these aren't an ABI in a way we care about in CheriBSD.

Reply:
Probably worth fixing AT_BSDFLAGS to not be sign-extended upstream independent of avoiding the issue like you've done?